shoop/simple_pricing/models.py

```diff
@@ -18,7 +18,6 @@ class SimpleProductPrice(models.Model):
     shop = models.ForeignKey("shoop.Shop", blank=True, null=True, db_index=True)
     group = models.ForeignKey("shoop.ContactGroup", blank=True, null=True, db_index=True)
```

Comment:
Shouldn't shop and group be changed to blank=False, null=False now, since shopless prices are not supported anymore and the groupless price is stored to ShopProduct.default_price?

That would also allow removing a few checks of group is null (like `exclude(group__isnull=True)`) in some other places.